docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,380 @@
+
+As a result, **MUI X v8 now supports Material UI v7 out-of-the-box**, ensuring your applications benefit from the latest improvements. We understand that change can be challenging, but migrating to v7 has been designed to be as smooth as possible—with comprehensive migration guides and GitHub support to assist you every step of the way.
+
+For users who want to upgrade to MUI X v8 while staying on Material UI v6, we've prepared a [guide](/x/migration/usage-with-material-ui-v5-v6/) to help you configure your project to keep using v6.
+
+## Migration Guide
+
+We have built a migration guide for Material UI and one for each of the advanced components. They detail every breaking change from previous versions with the recommended approach to fix them, ensuring that you can transition smoothly without the least amount of hassle. We care about an smooth migration, and that's been a priority for us when planning our new major versions.
+
+Please follow the instructions in our [migration guide](/x/migration/).
+
+## Long-Term Support
+
+We're now moving **MUI X v7 to Long-Term support**, while support for **MUI X v6 is discountinued**.
+In addition to critical bug fixes and security updates, v7 will continue to receive some bug fixes as a safe net for your your transition to MUI X v8.
+
+You can check our [supported versions in our docs](/x/introduction/support/#supported-versions).
+
+Once you’ve migrated to MUI X v8, you’ll benefit from the latest features and performance enhancements, with no disruptive breaking changes until the next major release.
+
+## What’s Next
+
+Our roadmap for MUI X remains ambitious. In the coming months, we’ll continue delivering significant improvements and feature enhancements, including:
+
+- **Charts and Data Grid integration**: Deeper interoperability for cohesive data visualization.
+- **Further improvements in the DX of the new Design Agnostic Data Grid**: We’ll continue iterating on the unstyled core to deliver a seamless and customizable experience across diverse design ecosystems.
+- **Further customization based on Composition**: Expanding our composable architecture for even more flexible customization.
+- **Increased leverage of AI**: Harnessing advanced AI to unlock new insights and streamline data analysis.
+- **Design and usability improvements**: Continuous refinements to enhance the overall user experience.
+- **Continue improvements on Server-Side integration**: Ongoing improvements for robust performance and scalability.
+
+## How to Get Involved
+
+Your input is vital to our continuous improvement. We invite you to:
```

Comment:
I don't love having three calls to action here, especially since two of them don't actually direct the reader anywhere.  I would suggest consolidating this final section into one clear CTA to get involved on GitHub.